website/docs/r/iam_trusted_profiles.html.markdown

```diff
@@ -0,0 +1,72 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profiles"
+description: |-
+  Manages iam_trusted_profiles.
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profiles
+
+Provides a resource for iam_trusted_profiles. This allows iam_trusted_profiles to be created, updated and deleted.
```

Comment:
Provides a resource for iam_trusted_profiles. This allows iam_trusted_profiles to be created, updated and deleted.

as
Create, modify, or delete IAM trusted profiles resources. . For more information, about IAM trusted profiles, see [provide the reference link of IAM trusted profiles]().